### PR TITLE
Wallet connect icon on modal

### PR DESCRIPTION
**Pull request by alongoni**
Solves #659 
image

### CODE REVIEW

File: src/theme/walletConnectModalCSS.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { css } from 'styled-components'
+
+export const reactWalletConnectModalCSS = css`
```
Comment (ramirotw) [inline src/theme/walletConnectModalCSS.tsx:3]: The only thing would be to remove the react part as the lib is just wallet connect. The rest is fine!

**Comment by elena-zh** [comment]
Could please someone create a pr-preview link for me to test these changes?
Thanks

File: src/theme/walletConnectModalCSS.tsx (hunk 1 above)
Comment (alongoni) [reply src/theme/walletConnectModalCSS.tsx]: Done! Thanks

**Review by ramirotw — APPROVED** [review]

**Comment by elena-zh** [comment]
LGTM

**Pull request merged**